package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-jsx-a11y": "^6.7.1",
+    "fontawesome": "^5.6.3",
```

Comment:
There's already a more recent font awesome library in this repo, see lines 13-17